src/Orchard/Orchard.Framework.csproj

```diff
@@ -419,7 +427,9 @@
     <Compile Include="Services\ClientHostAddressAccessor.cs" />
     <Compile Include="Services\DefaultJsonConverter.cs" />
     <Compile Include="Services\IClientHostAddressAccessor.cs" />
+    <Compile Include="Services\IYamlParser.cs" />
```

Comment:
We already have a Yaml parser that was made to fix performance issues. Isn't it sufficient to build this feature?